yaml/_type.ts

```diff
@@ -7,7 +7,25 @@
 import type { Any, ArrayObject } from "./_utils.ts";
 
 export type KindType = "sequence" | "scalar" | "mapping";
-export type StyleVariant = "lowercase" | "uppercase" | "camelcase" | "decimal";
+/**
+ * The style variation for `styles` option of {@linkcode stringify}
+ */
+export type StyleVariant =
+  | "lowercase"
+  | "uppercase"
+  | "camelcase"
+  | "decimal"
+  | "dec"
+  | 10
+  | "binary"
+  | "bin"
+  | 2
+  | "octal"
+  | "oct"
+  | 8
+  | "hexadecimal"
+  | "hex"
+  | 16;
 export type RepresentFn = (data: Any, style?: StyleVariant) => Any;
```

Comment:
Why do we have mutiple variants that mean the same thing? Is this behavior defined in the spec? Having multiple ways to do that same thing might confuse users.

Reply:
It's already implemented in int type. See `yaml/_type/int.ts`

Reply:
Oh, I see.